src/settings/components/account-settings/download-messages.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+/*
```

Comment:
duplicate licensing with different years, is it required ?

Reply:
nope